### PR TITLE
Removed api selector toggle from the settings menu

### PR DESCRIPTION
**Pull request by mpRegalado**
This PR removes the api selector toggle from the settings menu, since it was not actually affecting the state of the selector in the menu bar.

Closes https://github.com/wazuh/wazuh-kibana-app/issues/3335

### To test

- **Given** the browser is in wazuh app
- **When** the user navigates to `settings/configuration`
- **Then** the configuration panel does not show an api selector toggle

### Screenshots
Before
Before

After
After

### CODE REVIEW

File: public/react-services/app-state.js [hunk 1]
```diff
@@ -237,12 +237,6 @@ export class AppState {
    * Set a new value to the 'patternSelector' cookie
```
Comment (asteriscos) [inline public/react-services/app-state.js:237]: If the method was deleted it's comments should also be deleted.

**Review by CPAlejandro — APPROVED** [review]
CR: LGTM!
Testing: LGTM!

**Review by asteriscos — CHANGES_REQUESTED** [review]
Tested: :heavy_check_mark: 
Code looks good to me, except a small comment

**Review by asteriscos — APPROVED** [review]
LGTM

**Pull request merged**